### PR TITLE
 emoteclues: display when item is in inventory

### PR DESCRIPTION
**Pull request by devLotto**
**Red ✗** - item is not in inventory and is not equipped
**Yellow ✓** - item is in inventory, but not equipped
**Green ✓** - item is equipped/requirement is fulfilled

**Old:**

### CODE REVIEW

**Comment by MESLewis** [comment]
What if the yellow was a more neutral icon? Such as -

**Comment by iguaan** [comment]
Yellow checkmark sounds better in my opinion.

**Comment by devLotto** [comment]
Like this?

**Comment by MESLewis** [comment]
That looks perfect to me

**Comment by devLotto** [comment]
Updated

File: runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java [hunk 1]
```diff
@@ -609,25 +609,32 @@ public void makeOverlayHint(PanelComponent panelComponent, ClueScrollPlugin plug
...
-				boolean found = requirement.fulfilledBy(items);
+				boolean equipmentFulfilled = requirement.fulfilledBy(equipment);
+				boolean inventoryFulfilled = requirement.fulfilledBy(inventory);
 
 				panelComponent.getChildren().add(LineComponent.builder()
 					.left(requirement.getCollectiveName(plugin.getClient()))
 					.leftColor(TITLED_CONTENT_COLOR)
-					.right(found ? "X" : "-")
-					.rightColor(found ? Color.GREEN : Color.RED)
+					.right(equipmentFulfilled || inventoryFulfilled ? "✓" : "✗")
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java]: Can we possibly use unicode charcode here instead?

**Comment by devLotto** [comment]
Updated, now uses characters codes instead of the characters

**Pull request merged**